### ISSUE TITLE
FITS tables with astropy.table

### ISSUE THREAD

**Issue by astroAycha** [atpy/atpy#71]
Hello,

I am used to using ATpy to work with FITS tables and wanted to migrate to astropy.

But I am a bit confused though as the "migrating from ATpy to astropy" page: http://atpy.readthedocs.org/en/stable/migration.html
gives examples of reading and writing FITS tables with astropy.table, while the astropy doc page says that the FITS format is not supported yet.

When I try to read a FITS file with astropy.table, I get the following error message:
Exception: ('Format could not be identified. ', 'Valid formats are ascii, cds, daophot, hdf5, ipac, latex, rdb, sextractor, votable')

Cheers,
Aycha

**Comment by astrofrog** [comment]
@Labibahx - support for FITS in Astropy will only be available in 0.3 onwards. If you want you can switch to the latest developer version of Astropy:

```
git clone https://github.com/astropy/astropy.git
cd astropy
python setup.py install
```

Or you can wait a few weeks for the 0.3 release.

**Comment by astroAycha** [comment]
I see. 
Thanks @astrofrog . I will play with the dev version till the stable one is released :)
